### DISCUSSION TITLE
Installation via HACS and soldering mandatory?

### DISCUSSION THREAD

**Discussion by hartmood** [duhow/wibox#16] [Q&A] [answered]
Hi,

Is the soldering of the Wi-Box mandatory and secondly is there an option for a HACS installation available?

Best,
Harry

**Comment by duhow** [comment, marked as answer]
It is not "mandatory" to perform soldering, but it is highly recommended to set the pins in place and avoid any movements while using UART / serial.

HACS is not required since there is no custom integration in Home Assistant, but [MQTT integration](https://www.home-assistant.io/integrations/mqtt/) is needed.

**Reply by hartmood** [reply]
Thanks
